### PR TITLE
Resolve wire issues and remove difficult colors

### PR DESCRIPTION
**Pull request by neersighted**
:us: Remove British colors :us:

### CODE REVIEW

**Comment by Iamgoofball** [comment]
:us:

**Comment by neersighted** [comment]
Ah Travis, we hardly knew 'ye.

**Comment by GunHog** [comment]
american-eagle-flag-wallpaper-hd

**Comment by PKPenguin321** [comment]
:us: :gun: :us:
:+1: for freedom colors instead of degenerate colours

**Comment by ChangelingRain** [comment]
:uk:

**Comment by Anonmare** [comment]
:uk:
Down with the colonial scum tbh
:uk:

**Comment by optimumtact** [comment]
:gb: FOR QUEEN AND COUNTRY DENY THIS PR

**Comment by KazeEspada** [comment]
@optimumtact dont you luve in a former colony?

File: code/modules/admin/verbs/mapping.dm [hunk 1]
```diff
@@ -174,7 +174,7 @@ var/intercom_range_display_status = 0
 		if(isturf(thing))
 			continue
 		count++;
-	usr << "There are [count]/[MAX_FLAG] instances of non-turfs in the world."
+	usr << "There are [count]/65535 instances of non-turfs in the world."
```
Comment (RemieRichards) [inline code/modules/admin/verbs/mapping.dm]: this is incorrect anyways.

The following are limited to 16,777,216 instances at runtime:

/obj
/mob
/area
/image
Appearances (this is something internal with BYOND)
Unique overlay and underlay lists [This is an internal type called an ID array.]
User defined datums
[Consider turfs unofficially on this list.]

**Comment by RemieRichards** [comment]
See, @neersighted I didn't want you to just swap the numbers, because that's incorrect, you can have 16 mil turfs, and 16 mil PER OTHER TYPES, that's like, 7*16Mil or 112Mil non-turf instances.

**Comment by ike709** [comment]
image

**Pull request merged**